ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java

```diff
@@ -24,6 +24,8 @@
     private static final By ALLERGY_STATUS = By.cssSelector(".allergyStatus");
     private static final By ADD_NO_KNOWN = By.xpath("//*[contains(text(),'No Known Allergy')]");
     private static final By DELETE_NO_KNOWN = By.className("delete-action");
+    private static final By SELECT_NEW_ALLERGY = By.id("allergen-73667-label");  //codein
+    private static final By CONFIRM_SAVE_ALLERGY_BUTTON = By.id("addAllergyBtn");
```

Comment:
Which kind of Id is that

Reply:
> Which kind of Id is that

@sherrif10 can you try to inspect those `buttons` and check their ids